Framework/Core/include/Framework/DataOutputDirector.h

```diff
@@ -17,14 +17,13 @@
 #include "Framework/DataSpecUtils.h"
 #include "Framework/InputSpec.h"
 
-#include <regex>
+#include "rapidjson/document.h"
```

Comment:
Do you really need to expose it in a public header?

Reply:
what exactly is the problem with this?

Reply:
That we should try to avoid exposing unneeded dependencies, because it will bite us back. Can you use a `rapidjson/fwd.h` for example?

Reply:
done